plugins/Wallet/js/helpers.js

```diff
@@ -35,6 +30,15 @@ function hide(el) {
 		el.classList.add('hidden');
 	}
 }
+function eClass(name) {
+	return document.getElementsByClassName(name);
+}
+function classOnClick(name, reaction) {
+	var elements = eClass(name);
+	for (var i = 0; i < elements.length; i++) {
+		elements[i].onclick = reaction;
+	}
+}
```

Comment:
These should all have docstrongs that are more descriptive than // DOM Shortcuts